webui/react/__mocks__/determined-ui/kit/Tooltip.tsx

```diff
@@ -0,0 +1,11 @@
+import type { TooltipProps } from 'determined-ui/kit/Tooltip';
```

Comment:
Not crazy about `kit` here. When it was in the repo `kit` served to differentiate this code from the rest of the webui code but now that's done by the `determined-ui` label.

Reply:
done!